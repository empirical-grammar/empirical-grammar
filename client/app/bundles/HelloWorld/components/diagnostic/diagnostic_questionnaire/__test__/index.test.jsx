import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../index';

describe('<Index />',() =>{
    it('should render the childeren',()=>{
        const tree = renderer.create(
            <Index>Some Text</Index>).toJSON();
            expect(tree).toMatchSnapshot();
        
    });
});