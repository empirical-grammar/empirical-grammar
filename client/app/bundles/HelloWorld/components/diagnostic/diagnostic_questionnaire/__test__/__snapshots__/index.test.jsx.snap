// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should render the childeren 1`] = `
<div>
  Some Text
</div>
`;
